pages/runtime/resin-base-images.md

```diff
@@ -84,7 +84,7 @@ These are base images for different architectures:
 
 | Image | Arch | Repository | Available Tag|
 |:-----------|:------------|:------------|:------------|:------------|
```

Comment:
I think this needs to be changed to
```
|:-----------|:------------|:------------|:------------|
```
in order for GitHub to render this table correctly?

Reply:
@lurch it still renders 4 columns correctly even it should be 5. However, I've fixed this, thanks